components/brave_vpn/browser/connection/brave_vpn_region_data_helper.cc

```diff
@@ -94,6 +96,9 @@ mojom::RegionPtr GetRegionFromValueWithoutCity(const base::Value::Dict& value) {
   if (auto* name_pretty = value.FindString(brave_vpn::kRegionNamePrettyKey)) {
     region->name_pretty = *name_pretty;
   }
+  if (auto* country = value.FindString(brave_vpn::kRegionCountryKey)) {
+    region->country = *country;
```

Comment:
Curious what this data has and destkop also needs changes with this new property?

Reply:
It was a suggestion from the guardian team to persist the country and continent to use it for region selection to avoid future migration.

Reply:
`country` has country name? and then maybe desktop also use this property for country name?

Reply:
Ah, ok. I saw it from `region-list.ts`. It has country name.
I'll f/u to replace it with `namePretty` on desktop.